src/parser/jobs/drk/modules/MultiHitSkills.js

```diff
@@ -0,0 +1,65 @@
+import ACTIONS from 'data/ACTIONS'
+import Module from 'parser/core/Module'
+import {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import {Plural, Trans} from '@lingui/react'
+import {ActionLink} from 'components/ui/DbLink'
+import React from 'react'
+
+const REQUIRED_MULTI_HIT_TARGETS = {
+	[ACTIONS.EDGE_OF_SHADOW.id]: 2,
```

Comment:
ACTIONS.FLOOD_OF_SHADOW

Reply:
Whoops.  I had so much trouble keeping those two straight throughout the module.  Thanks for the catch, will fix shortly.